lib/transports/polling.js

```diff
@@ -237,7 +246,11 @@ Polling.prototype.send = function (packets) {
 
   var self = this;
   parser.encodePayload(packets, this.supportsBinary, function(data) {
-    self.write(data);
+    var compress = packets.some(function(packet) {
+      var options = packet.options || {};
```

Comment:
In other words, if we want a default for the public-facing API `.send()`, we set `compress: true` there.